ledger/accountdb.go

```diff
@@ -1131,8 +1305,6 @@ type baseAccountData struct {
 	TotalAppParams             uint64            `codec:"k"`
 	TotalAppLocalStates        uint64            `codec:"l"`
 
-	baseOnlineAccountData
-
```

Comment:
Bringing this back would make it easier to skip a catchpoint upgrade and avoid adding the new online account history table to the catchpoint merkle tree